_includes/footer.html

```diff
@@ -178,7 +162,7 @@ <h2 class="h6">
                         Washington, D.C. 20552
                     </p>
                     <div class="footer_general-inqueries">
-                        <b class="footer_general-inqueries-label">For general inquiries only</b>
+                        <b class="footer_general-inqueries-label"><i>For general inquiries only</i></b>
```

Comment:
Should we really be using `<b>` and `<i>`? It might be better to hook into the `.footer_general-inqueries-label` class.